modules/xmpp/JingleSessionPC.js

```diff
@@ -464,15 +465,15 @@ export default class JingleSessionPC extends JingleSession {
          * the value of RTCPeerConnection.iceConnectionState changes.
          */
         this.peerconnection.oniceconnectionstatechange = () => {
-            const now = window.performance.now();
             let isStable = false;
 
             if (!this.isP2P) {
-                this.room.connectionTimes[
-                    `ice.state.${this.peerconnection.iceConnectionState}`]
-                    = now;
+                window.performance.mark(`jitsi.ice.state.${this.peerconnection.iceConnectionState}`);
```

Comment:
Do we use this mark somewhere?

Reply:
Not really, but I didn't want to remove it. We may want to measure the time in between phases I guess? There is some of that already going on, but it's a bit involved and I didn't want to fall into that rabbit-hole :-P